packages/core/src/constants.ts

```diff
@@ -6,6 +6,9 @@ export const DATA_BUCKET_ATTR = 'data-make-styles-bucket';
 /** @internal */
 export const HASH_PREFIX = 'f';
 
+/** @internal */
+export const RESET_HASH_PREFIX = 'rf';
```

Comment:
What is the reason for the prefix to be 2 chars instead of just one?

Reply:
No reason, 8676f41